newsletter-november-2021.rst

```diff
@@ -0,0 +1,94 @@
+.. post:: November 4, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in November.
+
+Read the Docs newsletter - November 2021
+========================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- During the first week of November we attended the
+  2021 Essential Open Source Software for Science Annual Meeting,
+  an event organized by the CZI Science team.
+  We are thrilled to connect with projects in the Open Science ecosystem.
+- We upgraded our systems to Python 3.8 and Ubuntu 20.04 LTS.
+  This will allow us to leverage newer Python features
+  and use more up to date dependencies.
+- :doc:`The Read the Docs tutorial <readthedocs:tutorial/index>` is now complete!
+  :doc:`xxx </czi-grant-announcement>`
```

Comment:
nit: just a note to remember to update this before merging :)

Reply:
wooooooooooops changing, I forgot about this one